okhttp/src/main/java/okhttp3/Headers.java

```diff
@@ -234,7 +234,6 @@ public static Headers of(String... namesAndValues) {
   public static Headers of(Map<String, String> headers) {
     if (headers == null) throw new NullPointerException("headers == null");
 
-    // Make a defensive copy and clean it up.
```

Comment:
looks like this comment was accidentally copied from the implementation of `of(String...)`.